bot/seasons/evergreen/snakes/snakes_cog.py

```diff
@@ -278,9 +276,8 @@ def _snakify(message):
         return message
 
     async def _fetch(self, session, url, params=None):
-        """
-        Asyncronous web request helper method.
-        """
+        """Asyncronous web request helper method."""
```

Comment:
while here, might as well fix the typo:
```suggestion
        """Asynchronous web request helper method."""
```